.gitignore

```diff
@@ -38,5 +38,7 @@ prefabs.json
 *.diff
 *.sym
 *.js
+/pokeemerald-*.png
+/pokeemerald_modern-*.png
```

Comment:
We don't need this line because we don't append `_modern` at the end of modern. We could theoretically replace this with `_agbcc`, but then we should immediately just remove the line in upcoming